src/client/styles.css

```
@@ -148,3 +148,116 @@
+.dsh-better-markdown__user-bubble .dsh-better-markdown__markdown .markstream-react {
+  font: inherit;
+}
+
+/* Plain-text code blocks (empty fence or ```text) in the user bubble should
+ * wrap at the bubble edge instead of forcing horizontal scrolling. Markstream
+ * renders these as pre.code-fallback-plain with `white-space: pre` and
+ * `overflow: auto`; the Shiki path is handled below. */
+.dsh-better-markdown__user-bubble .markstream-react .code-fallback-plain {
+  white-space: pre-wrap !important;
+  overflow-wrap: anywhere !important;
+  word-break: break-word !important;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)